docs/docs/adrs/adr-020-cutomizable_slashing_and_jailing.md

```diff
@@ -0,0 +1,185 @@
+  If their validators are jailed, then the delegators can redelegate to other validators. 
+  Note though that delegators cannot redelegate multiple times, which means that if the new validators also get permanently jailed, the delegators need to wait for the unbonding period to elapse. 
+* **Testnet Consumer Chains.**
+  
+   ```go
+  double_sign.slash_fraction: 0 // no slashing
+  double_sign.jail_duration: 0 // no jailing
+  downtime.slash_fraction: 0 // no slashing
+  downtime.jail_duration: 0 // no jailing
+  ```
+
+  This means that validators are not punished for infractions on consumer chains. 
+  This setting is ideal for testing consumer chains before going in production, as neither validators nor their delegators incur any risk from the validators opting in on these consumer chains. 
+
+This means that both PoR and testnet consumer chains need only to cover the operational costs of the validators that opt in. 
+For example, if we take `$600` as the cost of running a validator node, a budget of `$3000` will be sufficient to cover the cost of four validators running such a consumer chain and have `$150` profit per validator as incentive.
+In practice, this can be implemented via the per-consumer-chain commission rate that allows validators to have different commission rates on different consumer chains. 
+
+### Implementation
+
+The implementation of this feature involves the following steps:
+
+* Add the `InfractionParameters` to `MsgCreateConsumer`.
+* On slashing events (for either downtime or double signing infractions), use the corresponding `slash_fraction` set by the consumer chain.
+* On jailing events (for either downtime or double signing infractions), use the corresponding `jail_duration` set by the consumer chain.
+* Cryptographic equivocation evidence received for PoR chains results in the misbehaving validators only being tombstoned and not slashed.
+* (Optional) Add the `InfractionParameters` to `MsgUpdateConsumer`, i.e., allow consumer chains to update the slashing and jailing parameters, but the changes will come into effect after a period equal to the staking module's unbonding period elapses to allow for validators to opt out.   
+
+## Consequences
+
+### Positive
+
+* Reduce the cost of ICS by removing the risk of slashing delegators.
+
+### Negative
+
+* Reduce the economical security of consumer chains with weaker slashing conditions.
+
+#### Economic Security Model without Slashing
+
+The economic security model of most Cosmos chains relies on the following properties:
+
+* validators are not anonymous, which means that they could be legally liable if they are malicious;
+* the delegated PoS mechanism creates a reputation-based network of validators;
+* most validators have most of their stake coming from delegations (i.e., nothing at stake, besides reputation);
+* it is relatively difficult to enter the active validator set and even more so to climb the voting power ladder.
```

Comment:
What about inactive validators?

Reply:
We could disable this for inactive validators. Or even better, users should choose their configuration.